.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18.x'
+      - name: Print versions
+        run: |
+          node --version
+          npm --version
+          npx commitlint --version
+      - name: Install commitlint
+        run: |
+          npm install -g @commitlint/cli @commitlint/config-conventional
+      - name: Configure commitlint
+        run: |
+          echo "module.exports = {extends: ['@commitlint/config-conventional']}" > commitlint.config.js
```

Comment:
There is so far no commitlint configuration, right?

Which types does it allow and which ones will it not accept? Or will it just accept anything that looks like a conventional commit, independent of type?

Reply:
The configuration enforces conventional commits : https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional#type-enum
So, only the types accepted in conventional commits will be accepted

Reply:
ok, but we have diverging ones in our convention. Compare also https://github.com/cloudfoundry/pcap-release/pull/81#discussion_r1259479864 (sorry, I thought I had submitted that already but it was still "pending")